src/main/java/seedu/duke/member/TeamMemberList.java

```diff
@@ -18,12 +18,6 @@ public static void removeTeamMember(String name) {
         memberList.removeIf(member -> (member.name.contains(name)));
```

Comment:
How do we handle deletion of members with duplicate names? One way could be not to allow duplicate members with same name to be created.

Reply:
Hmm then should we implement this for projects and tasks also?

Reply:
I think that output-ing a warning for the user and ask them to provide a full name or have them provide a specific index would be better.